src/components/ErrorModal.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div
+    class="modal-mask"
+    @click="resolve"
+  >
+    <div class="modal-wrapper" @click.stop>
+      <div class="error-modal">
+        <h1>{{ $t('reportBug') }}</h1>
+        <textarea
+          v-model="report"
+          class="report-text"
+        />
+        <button
+          class="button"
+          @click="send"
+        >
+          {{ $t('sendBugReport') }}
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { detect } from 'detect-browser';
+
+import SuccessIcon from '../assets/verifiedUrl.svg';
+import FailureIcon from '../assets/iconError.svg';
+import { backendFetch } from '../utils/backend';
+
+export default {
+  props: {
+    failure: { type: Boolean },
+    hideIcon: { type: Boolean },
+    resolve: { type: Function, required: true },
+    error: { type: [String, Number], default: 'Something went wrong' },
+  },
+  data() {
+    return {
+      SuccessIcon,
+      FailureIcon,
+      report: '',
+    };
+  },
+  methods: {
+    async send() {
+      const report = {
+        appVersion: process.env.npm_package_version,
+        browser: detect(),
+        error: this.error,
+        platform: 'web',
+        description: this.report,
+        time: new Date().toISOString(),
+      };
```

Comment:
Ensure that there is stacktrace in this report. Also, add `superhero-ui` somehow to distinguish it from wallet reports.

Reply:
Will do